micro_ros_common_diagnostics/package.xml

```diff
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>micro_ros_common_diagnostics</name>
-  <version>0.3.0</version>
+  <version>0.4.0</version>
```

Comment:
I see, I'm not fully aware of how bloom releases work, thanks. I did put it on a single commit to be sure I could easily roll it back just in case (which I'll do).
And yes, the reason why I bumped it is that it's a significant change